tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -265,6 +268,9 @@ public function test_get_settings_appearance_true_opts_in() {
 					'color'      => array(
 						'link' => true,
 					),
+					'dimensions' => array(
+						'minHeight' => true
```

Comment:
```suggestion
						'minHeight' => true,
```

Reply:
Thank you! We both noticed that at the same time 😄